include/trieste/driver.h

```diff
@@ -109,6 +109,13 @@ namespace trieste
       bool test_failfast = false;
       test->add_flag("-f,--failfast", test_failfast, "Stop on first failure");
 
+      std::optional<size_t> test_max_retries = std::nullopt;
+      test->add_option("-r,--max_retries", test_max_retries,
+                       "Maximum number of retries for finding unique trees");
+
+      auto entropy = test->add_subcommand("entropy",
```

Comment:
This is not expected to be a user facing feature?  I wonder if we should call it `debug_entropy`.  Perhaps add to the description that this is intended for debugging the fuzzing phase coverage rather than for normal usage of a tool based on Trieste?

Reply:
I think that is a good idea! Will fix